editor.planx.uk/src/@planx/components/List/schemas/GLA/CommunalSpace.ts

```diff
@@ -0,0 +1,36 @@
+import { Schema } from "@planx/components/List/model";
+
+export const CommunalSpaceGLA: Schema = {
+  type: "Unit of communal space",
+  fields: [
+    {
+      type: "question",
+      data: {
+        title: "Is this unit of communal space lost or gained?",
+        fn: "development",
+        options: [
+          { id: "gained", data: { text: "Gained", val: "gained" } },
+          { id: "lost", data: { text: "Lost", val: "lost" } },
+        ],
+      },
```

Comment:
my first go at summing identical units by development type was only accounting for the previous three options, but I'll update it to work the same when `gained` & `lost` are options too :+1:

Reply:
also nit picky thing: you use `loss` in later schemas, but `lost` here as "development" options - are they actually different in legislation or can we standardise?

Reply:
Thanks for spotting. loss/gain is what the data hub has in their technical schema across categories (including resi units), so I updated the lost/gained to loss/gain.